src/xo-app/vm/tab-disks.js

```diff
@@ -591,4 +632,4 @@ export default class TabDisks extends Component {
       </Row>
     </Container>
   }
-}
+  }
```

Comment:
Why are these changes also it this branch/PR?

Reply:
I have changed this just during viewing this tab while editing network tab of hosts.

Reply:
So this shouldn't be here, right?

Reply:
These changes are still here? :)

Reply:
Yes, it shouldn't be there, i redo it :)